src/tox.ini

```diff
@@ -1,53 +1,77 @@
-# Source charm: ./src/tox.ini
+# Classic charm: ./tox.ini
 # This file is managed centrally by release-tools and should not be modified
 # within individual charm repos.
 [tox]
-envlist = pep8
+envlist = pep8,py27,py35,py36
 skipsdist = True
+skip_missing_interpreters = True
```

Comment:
:-(

Reply:
+1 from me on all of the rest.  Please go ahead and land it when the tox ini items are addressed.  Thanks!  :cat2: